mypy.ini

```diff
@@ -70,6 +70,10 @@ ignore_errors = True
 
 ignore_errors = True
 
+[mypy-torchvision.models.detection.fcos]
+
+ignore_errors = True
```

Comment:
I wonder if that's necessary since you have all the types correct. I don't mind keeping it if it's not ready to be removed and tackle it along with the rest of the deteciton models.

Reply:
Hi @datumbox ,

The return type `Tuple[Dict[str, Tensor], List[Dict[str, Tensor]]]`  in [`eager_outputs`](https://github.com/xiaohu2015/vision/blob/e4970212ef6aa07e44498c045ee7cd6cccb3fe36/torchvision/models/detection/fcos.py#L406-L409) will lead to `mypy` failure, check the following for more details:

https://app.circleci.com/pipelines/github/pytorch/vision/12710/workflows/367d407f-e7a7-40c5-a691-30cab83a4322/jobs/1008230?invite=true#step-102-107

Reply:
Sounds good, thanks for pointing out. I agree, let's skip.